src/libstd/thread/local.rs

```diff
@@ -105,10 +105,12 @@ pub struct LocalKey<T> {
 #[allow_internal_unstable]
 macro_rules! thread_local {
     (static $name:ident: $t:ty = $init:expr) => (
+        #[cfg_attr(not(stage0), allow(static_could_be_const))]
         static $name: ::std::thread::LocalKey<$t> = {
             #[cfg_attr(all(any(target_os = "macos", target_os = "linux"),
                            not(target_arch = "aarch64")),
                        thread_local)]
+            #[cfg_attr(not(stage0), allow(static_could_be_const))]
```

Comment:
Wait, these ones shouldn't need the `allow`, AFAIK they always contain interior mutability.